src/components/shared/Performance.tsx

```diff
@@ -13,6 +14,7 @@ const useStyles = makeStyles<Theme, PerformanceProps, string>({
   performance: {
     fontSize: (props) => props.size || '24px',
     color: (props) => {
+      if (props.noPaint) return 'inherit';
```

Comment:
`'inherit'` has the same effect as not setting a color at all.

```suggestion
      if (props.noPaint) return undefined;
```

Reply:
I think 'inherit' is the better solution though from a semantic standpoint. It is clearer what happens.

Reply:
I get your point, but on the other hand there is no other CSS rule that's overwritten here. That's what 'inherit' is used for.

It's not that important and this should be refactored anyways.